example/src/components/Modal.tsx

```diff
@@ -0,0 +1,82 @@
+import { AdditionalColors } from '@colors';
+import { Typo } from '@components/Typo';
+import React, { ReactNode } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { ReactNativeModal } from 'react-native-modal';
+
+import { StandardButton, OnPressFunctionType } from './buttons/StandardButton';
+
+const ModalStyles = StyleSheet.create({
+  centeredView: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  modal: {
+    alignSelf: 'center',
+    width: 358,
+    justifyContent: 'center',
+    backgroundColor: AdditionalColors.white,
+    borderRadius: 16,
+  },
+  content: {
+    paddingLeft: 24,
+    paddingRight: 24,
+  },
+  header: {
+    marginTop: 32,
+  },
+  body: {
+    marginTop: 8,
+  },
+  topButton: {
+    marginTop: 40,
+  },
+  bottomButton: {
+    marginTop: 16,
+    marginBottom: 32,
+  },
+});
+
+type ModalProps = {
+  headline?: string;
+  body?: string;
+  visible: boolean;
+  children?: ReactNode;
+  onClose: OnPressFunctionType;
```

Comment:
```suggestion
  onClose: () => void;
```

Reply:
We don't need `GestureResponderEvent`, we just want to know when user dismissed the modal.

Reply:
Done.